src/redux/reducers/localStorage-reducer.js

```diff
@@ -0,0 +1,21 @@
+import localStorageActions from "redux/actions/localStorage-actions";
+
+const initialState = {
+  items: []
+}
+
+const LocalStorageReducer = (state = initialState, action) => {
+  const {items} = state;
+  const {type, payload} = action;
+  switch (type) {
+    case localStorageActions.ADD:
+      return {
+        ...state,
+        items: [payload, ...items]
+      };
+    default:
+      return state;
+  }
+};
+
+export default LocalStorageReducer;
```

Comment:
This file is good, lets name it something other than LocalStorage though. 

Let's name it "UploadHistory"